plugins/destination/gcs/client/spec/schema_test.go

```diff
@@ -0,0 +1,199 @@
+package spec
+
+import (
+	"testing"
+
+	"github.com/cloudquery/codegen/jsonschema"
+)
+
+func TestSpecJSONSchema(t *testing.T) {
+	// cases about embedded filetypes.FileSpec are tested in the corresponding package
+	// However, we add some tests to verify that it actually is properly working
+	jsonschema.TestJSONSchema(t, JSONSchema, []jsonschema.TestCase{
+		{
+			Name: "csv file spec",
+			Spec: `{
+	"format": "csv",
+	"format_spec": {
+		"skip_header": true,
+		"delimiter": "#"
+	},
+	"path": "abc",
+    "bucket": "abc"
+}`,
```

Comment:
erm, indent?

Reply:
it doesn't matter here + we use the raw string